server/src/utils/mixerConnections/SSLMixerConnection.ts

```diff
@@ -549,22 +505,7 @@ export class SSLMixerConnection {
     }
 
     updateChannelName(channelIndex: number) {

```

Comment:
This method looks like it should be deleted.

All it does is return true. Where this method is called we might as well just give it the 'true' value directly.

The method name is completely misrepresenting what the method does. `updateChannelIndex` indicates a update will happen when this method is called. No updates is happening, so the method needs a new name. (Ideally it should be deted)

You should be able to look at a methods name (and nothing else) and know what the method does. If it's not possible to come up with a name for a method like that, then that's an indication that the method is too complex or is having multiple responsibilities (and it should only have one responsibility).